04-store-api/starter/controllers/products.js

```diff
@@ -0,0 +1,97 @@
+const Product = require('../models/product')
+const queryValidator = require('../utils/queryValidator')
+
+const DEFAULT_LIMIT = 10;
```

Comment:
Great use of constants!  Configuration parameters should be visible near the top, just like this!